src/AppScopeTrait.php

```diff
@@ -3,5 +3,43 @@
 namespace atk4\core;
 
 trait AppScopeTrait {
+
+    public $_appScope = true;
```

Comment:
For consistency correct name of this property would be `$_appScopeTrait = true`. Equal with Trait class name started with underscore and first letter lowercased. Other traits do like this already.